official/mnist/dataset.py

```diff
@@ -52,7 +52,7 @@ def check_labels_file_header(filename):
   """Validate that filename corresponds to labels for the MNIST dataset."""
   with tf.gfile.Open(filename, 'rb') as f:
     magic = read32(f)
-    num_items = read32(f)
+    num_items = read32(f)  # pylint: disable=unused-variable
```

Comment:
I think instead of put a suppression of lint here, you can just rename the variable as unused_num_items and I think it will make the code more readable, and also make the lint warning go away.

Reply:
Solved slightly differently, as per @asimshankar 's comments above.